funnel/forms/participant.py

```diff
@@ -26,3 +26,28 @@ class ParticipantForm(forms.Form):
 class ParticipantBadgeForm(forms.Form):
     choices = [('', "Badge printing status"), ('t', "Printed"), ('f', "Not printed")]
     badge_printed = forms.SelectField("", choices=[(val_title[0], val_title[1]) for val_title in choices])
+
+
+# TODO: Move to Baseframe
+class ValidFile(object):
+    def __init__(self, allowed_exts=[], message=None):
```

Comment:
You need a way to signal that all extensions are allowed. Check for an empty list below.